che-editors.yaml

```diff
@@ -109,7 +109,7 @@ editors:
         volume: {}
       - name: che-machine-exec
         container:
-          image: 'quay.io/eclipse/che-machine-exec:nightly'
+          image: 'quay.io/eclipse/che-machine-exec:next'
```

Comment:
Has this change already been made in the che-machine-exec side of things?

Reply:
yes, machine exec is now being published along with nightly